starter-code/functions-and-arrays.js

```diff
@@ -43,7 +58,13 @@ console.log(average);
 
 // Array of Strings
 function averageWordLength (array) {
```

Comment:
This is the solution I expected. Reusing code is almost always the best move. Keep it up!